src/renderers/WebGLRenderer.js

```diff
@@ -2885,6 +2892,17 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 	}
 
+	function getTextureTarget ( texture ) {
```

Comment:
Hmm the only thing is that `texture.type` refers to unsigned byte / float / etc. Whereas "target" is used in GL spec to mean `gl.TEXTURE_2D` vs `gl.TEXTURE_3D` etc.